docs/source/tutorials/ontologies.rst

```diff
@@ -0,0 +1,26 @@
+********
+Ontologies
+********
+
+A central component of a project is the ontology - an in-depth description of which can be found `here <https://docs.encord.com/ontologies/overview>`_.
+
+The ontology essentially defines the label structure of a given project.
+
+- For a platform description of the ontology, please see :xref:`configure_label_editor_(ontology)`.
+- To access the ontology you can use the :meth:`~encord.user_client.EncordUserClient.get_ontology()` method.
+    - This allows you to work with its `structure` property to read or write parts of the ontology.
```

Comment:
When you say `to read or write parts of the ontology` it might be confusing. The `structure`, is not the ontology in this sense I guess. I know these terms are overloaded, but I think we need to be careful on how we use these terms not to add more confusion. 
I hope that somewhat makes sense.